benchmarks/planetarium_plot_methods.cpp

```diff
@@ -58,49 +64,45 @@ constexpr Length far = 400'000 * Kilo(Metre);
 constexpr Length focal = 1 * Metre;
 
 Perspective<Navigation, Camera> PolarPerspective(
+    Similarity<Navigation, GCRS> const& navigation_to_gcrs_at_epoch,
     Length const distance_from_earth) {
-  using LeftNavigation =
-      Frame<struct LeftNavigationTag, Arbitrary, Handedness::Left>;
-  return {
-      RigidTransformation<Navigation, Camera>(
-          Navigation::origin + Displacement<Navigation>(
-                                   {0 * Metre, 0 * Metre, distance_from_earth}),
-          Camera::origin,
-          Rotation<LeftNavigation, Camera>(
-              Vector<double, LeftNavigation>({1, 0, 0}),
-              Vector<double, LeftNavigation>({0, -1, 0}),
-              Bivector<double, LeftNavigation>({0, 0, -1}))
-                  .Forget<OrthogonalMap>() *
-              Signature<Navigation, LeftNavigation>(
-                  Sign::Positive(),
-                  Sign::Positive(),
-                  DeduceSignReversingOrientation{})
-                  .Forget<OrthogonalMap>())
-          .Forget<Similarity>(),
-      focal};
+  using LeftGCRS = Frame<struct LeftGCRSTag, Arbitrary, Handedness::Left>;
```

Comment:
Is there a reason for using left-handed frames?

Reply:
Probably that we do so in-game.